client/src/Services/fetchData.test.js

```diff
@@ -0,0 +1,14 @@
+import fetchData from './fetchData';
+
+describe('fetchData', () => {
+  it('returns an object and calls fetch twice', async () => {
```

Comment:
I looked at `fetchData` and thought it should be renamed, because it is really specific, always fetching two specific things. Also, it looks like you only added the parameter, so you can check in this test whether the function has been called or not. It might be cleaner to use [Jest's "mocking modules" feature](https://jestjs.io/docs/en/mock-functions#mocking-modules). Maybe in a separate PR if it holds you back from doing other things, but definitely consider it please